src/Android/Services/AndroidPushNotificationService.cs

```diff
@@ -52,6 +55,31 @@ public Task UnregisterAsync()
             // Do we ever need to unregister?
             return Task.FromResult(0);
         }
+
+        public void DismissLocalNotification(string notificationId)
+        {
+            var notificationManager = NotificationManagerCompat.From(Android.App.Application.Context);
+            notificationManager.Cancel(int.Parse(notificationId));
+        }
```

Comment:
I'd add a try parse check here to avoid potential future crashes because if the `notificationId` is `null` or something that is not an `int` here the Parse will crash:
```c#
public void DismissLocalNotification(string notificationId)
{
    if (int.TryParse(notificationId, out int intNotificationId)
    {
        var notificationManager = NotificationManagerCompat.From(Android.App.Application.Context);
        notificationManager.Cancel(intNotificationId);
    }
}
```